README.md

```diff
@@ -8,10 +8,13 @@
 The SPIR-V Tools project provides an API and commands for processing SPIR-V
 modules.
 
-The project includes an assembler, binary module parser, disassembler, and
-validator for SPIR-V, all based on a common static library. The library contains
-all of the implementation details, and is used in the standalone tools whilst
-also enabling integration into other code bases directly.
+The project includes an assembler, binary module parser, disassembler,
+validator, and optimizer for SPIR-V. Except for the optimizer, all are based
+on a common static library.  The library contains all of the implementation
+details, and is used in the standalone tools whilst also enabling integration
+into other code bases directly. The optimizer implementation resides in its
+own library, but also depends on the common library for SPIR-V bilateral
```

Comment:
Skip the last phrase "for SPIR-V bilateral text to binary conversions".  It's detail they don't need at this point.  Also, it will get stale over time.